Headers/Foundation/NSObjCRuntime.h

```diff
@@ -49,6 +49,19 @@
 #define PRIuPTR "lu"
 #endif
 
+/* MINGW 32/64 go here... */
+#if defined(__MINGW32__)
+/* NONE FOR NOW... */
+#endif
+#if defined(__MINGW64__)
+#undef PRIuPTR
+#define PRIuPTR "llu"
```

Comment:
Is this right?  I would expect the format for a pointer-length unsigned integer to be lu rather than llu on 64bit.